compliance_suite/cli.py

```diff
@@ -42,7 +41,7 @@ def main():
 
 
 @main.command(help='run compliance utility report using base urls')
-@click.option('--server', '-s', multiple=True, help='base_url')
+@click.option('--server', '-s', multiple=False, help='base_url')
```

Comment:
This will disable the possibility of creating the report for multiple server. 
Is it intended ?

Reply:
I recall doing this change with Jeremy but dont remember the reason. I will do some testing with the testbed changes to see if it supports multiple servers and get back to this.